Bowling.java

```diff
@@ -87,9 +92,8 @@ public void update(Integer pinsDown) {
 		Frame currentFrame = getCurrentFrame();
 		currentFrame.update(pinsDown);
 
-		if (currentFrame.isStrike()) {
+		if (currentFrame.isStrike())
```

Comment:
When you bowl a strike, you knock 10 pins down on the first bowl and then your turn is over. You don't get a second bowl. So `currentFrame.update(0)` adds a 0 for `pinsDown` for that second bowl, which forces the frame to be `isFull` so that an additional frame is added for the next turn. Scoreboard: [10,0], []

When you bowl a spare, this is not needed because you knock all 10 pins down between both the first and second bowls. In this case, the frame `isFull` and an additional frame is added automatically. Scoreboard: [7,3], []

Does this make sense to get to the next frame after bowling a strike this way?